src/datachain/error.py

```diff
@@ -1,15 +1,3 @@
-import botocore.errorfactory
```

Comment:
This increases import time, so I am removing this and reraising any unhandled error as `ClientError`.